iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/CalculatedBusImpl.java

```diff
@@ -188,6 +188,23 @@ public Component getSynchronousComponent() {
         return terminalRef == null ? null : scm.getComponent(terminalRef.getSynchronousComponentNumber());
     }
 
+    @Override
+    public VoltageLimits getVoltageLimits() {
+        return getConnectableStream(BusbarSection.class)
+                .map(VoltageLimitsHolder::getVoltageLimits)
+                .filter(Objects::nonNull)
+                .findFirst() // we suppose that all voltage limits in a same calculated bus are equals
```

Comment:
I'm pretty sure it won't be the case: I propose to sort the voltageLimits and return the lowest to avoid missing violations